tonic/src/metadata/mod.rs

```diff
@@ -29,6 +29,8 @@ pub use self::value::AsciiMetadataValue;
 pub use self::value::BinaryMetadataValue;
 pub use self::value::MetadataValue;
 
+pub(crate) use self::map::GRPC_TIMEOUT_HEADER;
```

Comment:
Do we need to add an allow here?

Reply:
I gave it a `#[cfg(feature = "transport")]`